platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/protocol/SyncProtocol.java

```diff
@@ -46,6 +49,17 @@
  * This object will be instantiated once per peer, and is bidirectional
  */
 public class SyncProtocol implements Protocol {
+    /**
+     * The platform statuses that permit syncing. If the platform isn't in one of these statuses, no syncs will be
+     * initiated or accepted
+     */
+    public static final Collection<PlatformStatus> STATUSES_THAT_PERMIT_SYNC = Set.of(
```

Comment:
Can we have a separate class that encapsulates this logic? That way, the sync protocol only does `shouldSync()`